src/graphql/mirror.js

```diff
@@ -318,6 +318,18 @@ export class Mirror {
           CREATE INDEX idx_connection_entries__connection_id
           ON connection_entries (connection_id)
         `,
+        dedent`\
+          CREATE TABLE network_log (
+              rowid INTEGER PRIMARY KEY,
+              query TEXT NOT NULL,
+              query_parameters TEXT NOT NULL, -- stringified JSON object
+              request_timestamp INTEGER,
```

Comment:
I think `request_timestamp` should be `NOT NULL`?

Reply:
Just noticed https://github.com/sourcecred/sourcecred/pull/1562#discussion_r367784945 this was already discussed. Go ahead and ignore. :smile: